Extension.php

```diff
@@ -15,6 +15,18 @@ public function initialize()
         $this->app->before(array($this, 'before'));
 
         /*
+         * Own Twig functions:
+         * You can define methods inside this class as Twig functions.
+         * How to use the following example in your template: {{ addFiveTo(7) }}
+         * It will display '12'.
+         * */
+
+        $this->addTwigFunction(
+            'addFiveTo', // Twig function name
+            'addFiveTo'  // Method in this class (scroll down buddy)
+        );
+
```

Comment:
Just to make things confusing, Twig standard is lower case function names, with snake case as required :confused: 

So something more like:
```php
        $this->addTwigFunction(
            'add_five_to', // Twig function name {{ add_five_to() }}
            'addFiveTo'    // Method name in this class 
        );
```